web/src/components/customer/listing-details/DeliveryDetailsPrompt.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import {useFormPropsContext} from 'components/common/contexts/FormPropsContext';
+import MobilePrompt from 'components/common/MobilePrompt';
+import {useDeliveryDetailsPromptContext} from 'components/customer/listing-details/contexts/DeliveryDetailsPromptContext';
+import MobileFormRow from 'components/customer/listing-details/MobileFormRow';
+
+import {ReactComponent as DeliverySvg} from 'assets/customer/delivery.svg';
+
+/**
+ * DeliveryDetailsPrompt that shows the delivery details prompt.
```

Comment:
Let's stick with the word "fulfillment" instead of "delivery" to refer to the stage when merchant is trying to fulfill a customer's demand, being a product getting delivered or a service being offered.

Reply:
@xlai20 Hmm this prompt is specifically asking for delivery details of the customer though, so I thought it should be good to be more specific for frontend components instead of `FulfilmentDetailsPrompt` which sounds more vague. As you said, "fulfilment" might refer to a service, but that's not really what the prompt and form is dealing with. But if you think `FulfilmentDetailsPrompt` I can just rename it!

Reply:
Yup I do think fulfillment is a better name as it comprises both service and physical item delivery. Also more consistent with Karen's naming convention on merchant side. Nevertheless, if renaming is difficult then you can just proceed with the current name. This one is trivial.